.buildkite/pipeline.yml

```diff
@@ -146,11 +146,11 @@ steps:
         echo "--- :arrow_up: Upload Android bundle artifact"
         buildkite-agent artifact upload bundle/android/App.js
 
-        if [[ -n "$BUILDKITE_TAG" ]]; then
+        # if [[ -n "$BUILDKITE_TAG" ]]; then
```

Comment:
I needed to generate the source map artifacts in order to upload them to Sentry. For reference, this is the command I used:
```
sentry-cli releases -o a8c -p jetpack-ios files "com.jetpack.alpha@24.2+pr22654-07765b3" upload-sourcemaps \
--dist "pr22654-07765b3" \
--strip-common-prefix \
--bundle <LOCAL_PATH>/main.jsbundle \
--bundle-sourcemap .<LOCAL_PATH>/main.jsbundle.map
```

Note that the `pr22654-07765b3` part needs to be updated based on the installable build to test (https://github.com/wordpress-mobile/WordPress-iOS/pull/22654#issuecomment-1956303481). Same for `<LOCAL_PATH>` which is based on where the bundle and source map are located.